packages/rainbowkit/src/components/ConnectOptions/MobileOptions.tsx

```diff
@@ -100,7 +102,11 @@ enum MobileWalletStep {
 export function MobileOptions({ onClose }: { onClose: () => void }) {
   const titleId = 'rk_connect_title';
   const wallets = useWalletConnectors();
-  const { learnMoreUrl } = useContext(AppContext);
+  const { disclaimer, learnMoreUrl } = useContext(AppContext);
+  const disclaimerComponent = disclaimer?.({
```

Comment:
Conditionally calling it as a function can cause issues if their component uses hooks. We should render it via JSX instead.